pkgs/by-name/ap/apple-sdk/common/process-stubs.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  jq,
+  libtapi,
+}:
+
+let
+  removedDylibs = [
+    # Without headers, this isn’t very useful. Headers can’t be distributed.
+    "/usr/lib/system/libcorecrypto.dylib"
+  ];
```

Comment:
I’m removing the corecrypto stubs from the SDK because the corecrypto license is non-free, can’t use.

There is corecrypto source and headers included with xnu. It has both the non-free corecrypto license and APSL 2.0, but the latter seems like a mistake.